portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACTimeFilter.path

```diff
@@ -0,0 +1,60 @@
+<html>
+<head>
+  <title>ACTimeFilter</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+  <thead>
+  <tr><td rowspan="1" colspan="3">ACTimeFilter</td></tr>
+  </thead>
+  <tbody>
+  <!-- TIME FILTER ELEMENTS  -->
+  <tr>
+    <td>TIME_FILTER_DROPDOWN</td>
+    <td>//div[@class='dropdown dropdown-range-key-root']/button</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>CARD_TIME_FILTER_DROPDOWN</td>
+    <td>(//h5[@class='card-title' and text()='${key_cardName}']/following::div[@class='dropdown dropdown-range-key-root']/button)[1]</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>TIME_FILTER_DROPDOWN_OPTION</td>
```

Comment:
I would refactor using the _contains strategy_ like:

`//div[contains(@class,'dropdown-menu') and contains (@class,'show')]//button[text()='More Preset Periods']`